templates/addons/storageclass-azure-disk.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: storage.k8s.io/v1beta1
```

Comment:
I don't see this being used in the script above. If it is independent, let's split the commit in 2 commits (same PR, just two commit messages).